### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rhcp89**

### CODE REVIEW

**Comment by rhcp89** [comment]
@ildevelop small fix

File: js/node-express-postgress-sequelize/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Sequelize, Node.js, Express and PostgreSQL RESTful API
 
-Using the [DOC](https://docs.metisdata.io/metis/sdk-integration/javascript-sequelize) of [Metis](https://app.metisdata.io/) and this source code is part of [Node.js, Express.js, Sequelize.js and PostgreSQL RESTful API](https://www.djamware.com/post/5b56a6cc80aca707dd4f65a9/nodejs-expressjs-sequelizejs-and-postgresql-restful-api) tutorial.
+Using the [Documentation](https://docs.metisdata.io/metis/sdk-integration/javascript-sequelize) of [Metis](https://app.metisdata.io/) and this source code is part of [Node.js, Express.js, Sequelize.js and PostgreSQL RESTful API](https://www.djamware.com/post/5b56a6cc80aca707dd4f65a9/nodejs-expressjs-sequelizejs-and-postgresql-restful-api) tutorial.
```
Comment (ildevelop) [inline js/node-express-postgress-sequelize/README.md:5]: Screen Shot 2022-12-08 at 15 48 02

File: js/node-express-postgress-sequelize/README.md [hunk 2]
```diff
@@ -11,28 +11,32 @@ Insert your api key into `METIS_API_KEY` at `./models/index.js line:43`
 
 ### 2. Setup a local PostgreSQL Database.
 
-<ins>Option 1</ins>: Native PostgreSLQ
+<ins><b>Option 1</b></ins>: Native PostgreSLQ
```
Comment (ildevelop) [inline js/node-express-postgress-sequelize/README.md:14]: you can use 2* some texe 2* -> **some text**

**Review by ildevelop — APPROVED** [review]

File: js/node-express-postgress-sequelize/README.md (hunk 1 above)
Comment (rhcp89) [reply js/node-express-postgress-sequelize/README.md:5]: This one is fine imo.

File: js/node-express-postgress-sequelize/README.md (hunk 2 above)
Comment (rhcp89) [reply js/node-express-postgress-sequelize/README.md:14]: fixing

**Pull request merged**